src/MAUI/Maui.Samples/Samples/Map/SetBasemap/readme.metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+    "category": "Map",
+    "description": "Change a map's basemap.",
+    "formal_name": "SetBasemap",
+    "ignore": false,
+    "images": [
+        "setbasemap.jpg"
+    ],
+    "keywords": [
+        "basemap",
+        "basemap gallery",
+        "basemap style",
+        "map",
+        "toolkit"
+  ],
+    "offline_data": [],
+    "redirect_from": [
+        "/net/maui/sample-code/set-basemap.htm"
```

Comment:
Can you add redirect link from `/net/maui/sample-code/change-basemap.htm`? And this for all platforms? Otherwise, looks great.